tests/all/js_globals/WebAssembly.rs

```diff
@@ -0,0 +1,30 @@
+#![allow(non_snake_case)]
+
+use super::project;
+
+#[test]
+fn validate() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use JsValue;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js::WebAssembly;
+
+            #[wasm_bindgen]
+            pub fn validate_wasm(wasm: JsValue) -> bool {
+                WebAssembly::validate(wasm).unwrap_or(false)
+            }
+        "#)
+        .file("test.ts", r#"
```

Comment:
⚠️ This needs to be `test.js` to become green on travis before we've addressed [this](https://github.com/rustwasm/wasm-bindgen/blob/efa4a2b8fa5da3e5aecd1a71f060c1954fc58cf4/tests/all/project_builder.rs#L233).